app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -1210,7 +1210,7 @@ class HomeFragment : Fragment() {
         return shouldEnableWallpaper() && settings.shouldAnimateFirefoxLogo &&
             onboarding.userHasBeenOnboarded() &&
             settings.numberOfAppLaunches >= 3 &&
-            FeatureFlags.isThemedWallpapersFeatureEnabled(localContext)
+            FeatureFlags.showWallpapers
```

Comment:
I think we could remove it, the check it's already part of `shouldEnableWallpaper` :)